dev/build_release.py

```diff
@@ -231,6 +250,83 @@ def start_deb_build(self, name):
       logfile='{name}-debian-build.log'.format(name=name)
     )
 
+  def start_rpm_build(self, name):
+    """Start a subprocess to build and publish the designated component.
+
+    This function runs a gradle 'buildRpm' task using the last git tag as the
+    package version and the Bintray configuration passed through arguments. The
+    'buildRpm' task release builds the source, packages the redhat and jar
+    files, and publishes those to the respective Bintray '$org/$repository'.
+
+    The naming of the gradle task is a bit unfortunate because of the
+    terminology used in the Spinnaker product release process. The artifacts
+    produced by this script are not 'release candidate' artifacts, they are
+    pre-validation artifacts. Maybe we can modify the task name at some point.
+
+    The gradle 'buildRpm' task throws a 409 if the package we are trying to
+    publish already exists. We'll publish unique package versions using build
+    numbers. These will be transparent to end users since the only meaningful
+    version is the Spinnaker product version.
+
+    We will use -Prelease.useLastTag=true and ensure the last git tag is the
+    version we want to use. This tag has to be of the form 'X.Y.Z-$build' or
+    'vX.Y.Z-$build for gradle to use the tag as the version. This script will
+    assume that the source has been properly tagged to use the latest tag as the
+    package version for each component.
+
+    Args:
+      name [string]: Name of the subsystem repository.
+
+    Returns:
+      BackgroundProcess
+    """
+    jarRepo = self.__options.jar_repo
+    parts = self.__options.bintray_repo.split('/')
+    if len(parts) != 2:
+      raise ValueError(
+          'Expected --bintray_repo to be in the form <owner>/<repo>')
+    org, packageRepo = parts[0], parts[1]
+    bintray_key = os.environ['BINTRAY_KEY']
+    bintray_user = os.environ['BINTRAY_USER']
+
+    if self.__options.nebula:
+      target = 'buildRpm'
+      extra_args = [
+          '--stacktrace',
+          '-Prelease.useLastTag=true',
+          '-PbintrayPackageBuildNumber={number}'.format(
+              number=self.__build_number),
+          '-PbintrayOrg="{org}"'.format(org=org),
```

Comment:
I think the `-Pbintray*` properties aren't required here if we're not doing the upload via gradle.